code/2_analysis/maps/figA4a.R

```diff
@@ -0,0 +1,63 @@
+# load crop productivity data
+grace_croppy <- read_dta("data/intermediate/grace_merged/crop_spec_pot_yld.dta")
+
+# imitate the "bi_class" function as it does not work for some reason here
+bisc_rice_prod <- grace_croppy %>% 
```

Comment:
odd!! any idea why you had to reinvent the wheel on this one but not others?

Reply:
I looked into it, and I believe it was because the underlying `bi_class` function uses the base function `cut`, but the function isn't flexible enough to handle repeated breaks. For rice in particular, the minimum potential yield is the same as the first quartile. That's why the function errors out.